doc/api/diagnostics_channel.md

```diff
@@ -227,6 +227,56 @@ diagnostics_channel.subscribe('my-channel', onMessage);
 diagnostics_channel.unsubscribe('my-channel', onMessage);
 ```
 
+#### `diagnostics_channel.tracingChannel(nameOrChannels)`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `nameOrChannels` {string|TracingChannel} Channel name or
+  object containing all the [TracingChannel Channels][]
```

Comment:
```suggestion
  array containing all the [TracingChannel Channels][]
```

Shouldn't the type be: `{string|TracingChannel|TracingChannel[]}`?

Reply:
No, it doesn't take an array, it takes an object mapping names to the corresponding channels of a TracingChannel. So you could use custom channels for each of the five channels (`start`, `end`, `asyncStart`, `asyncEnd`, `error`) rather than the defaults. The `TracingChannel` type here refers to it not as a class but as an object shape to copy into a _new_ TracingChannel.